python/pyspark/ml/classification.py

```diff
@@ -286,56 +286,48 @@ def setParams(self, featuresCol="features", labelCol="label", predictionCol="pre
     def _create_model(self, java_model):
         return LinearSVCModel(java_model)
 
-    @since("2.2.0")
```

Comment:
This is the correct version, isn't it? If this is correct, it seems that we don't need to remove.
> In docs of all the previous versions, setMaxIter in LinearSVC doesn't have version info.

Reply:
Yes. The versions are correct.